app/services/search/book_versions/index.rb

```diff
@@ -39,14 +38,25 @@ def recreate
     end
 
     def delete
-      OpenSearch::ElasticsearchClient.instance.indices.delete(index: name)
+      if OpenSearch::ElasticsearchClient.instance.indices.exists?(index: name)
```

Comment:
What happens if doesn't exist and try to delete it?  If it doesn't do anything bad, we can take this `exists?` call out.  If it does raise an exception, let's change to making the call and rescuing the exception, because as is there's a very-rare chance that the index could be deleted between the `exists?` and `delete` calls anyway.

Reply:
it does raise an exception.